docs/developers/Building_Android.md

```diff
@@ -68,3 +68,17 @@ cmake --build ../build
 ```
 
 You can also see a more detailed walkthrough on CMake configuration at [How to build VCMI (macOS)](./Building_macOS.md).
+
+## Docker
+
+For developing it's also possible to use docker to build an android APK. As requirement only a installed docker is needed. The container image contains all other prerequisites.
+
+To build using docker just open a terminal with `vcmi` as working directory.
+
+Build the image with (only needed once):
+`docker build -f docker/BuildAndroid.dockerfile -t vcmi-android-build .`
+
+After building the image you can compile vcmi with:
+`docker run -it --rm -v $PWD/:/vcmi vcmi-android-build`
+
+The current dockerfile is ARM64 only but can adjusted manually for ARM32.
```

Comment:
please reflect it in the name of the dockerfile. Also, correct names are: armv7 for ARM32 and aarch64 for ARM64.